client/packages/system/src/Stock/DetailView/Footer.tsx

```diff
@@ -44,9 +48,10 @@ export const Footer: FC<FooterProps> = ({
             marginLeft="auto"
           >
             <DialogButton
-              variant="cancel"
-              disabled={disabled}
-              onClick={() => showCancelConfirmation()}
+              variant="close"
+              onClick={() =>
+                isDirty ? showCancelConfirmation() : navigateUpOne()
```

Comment:
with the change - can you please add the `navigateUpOne` to the `onConfirm` of the cancel confirmation modal. The button now says 'close' but when you click it, effectively nothing happens. The `resetDraft` call doesn't revert my changes and I'm still on the same page!

Side note: the resetDraft should be doing something, right? and with this change it can actually be removed - we don't have the pattern of resetting form state anywhere else in the app

```
  const onCancel = () => {
    resetDraft();
    navigateUpOne();
  };

  const showCancelConfirmation = useConfirmationModal({
    onConfirm: onCancel,
    message: t('messages.confirm-cancel-generic'),
    title: t('heading.are-you-sure'),
  });
```

Reply:
The resetDraft resets draft 🤣

Reply:
buut, I edited, clicked the button and my changes didn't get reset 🤷